Assets/Scripts/Bomb.cs

```diff
@@ -2,15 +2,20 @@
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.Tilemaps;
+using System;
 
 public class Bomb: MonoBehaviour
 {
     // projectile script values
     public float detonateTime = 1;
     public float cameraShakeDuration = 0.2f;
     public float explosionDuration = 0.2f;
+    public float explosionMultiplier = 1.0f;
```

Comment:
What is this exactly? The number of squares each?

Reply:
Kind.  It scales the explosion in the X and Y direction.   Ill add a comment